bindings/c/Cargo.toml

```diff
@@ -13,6 +13,9 @@ cbindgen = "0.24.0"
 [dependencies]
 bytes = "1.5.0"
 lazy_static = "1.4.0"
-libsql = { path = "../../libsql", features = ["encryption"] }
+libsql = { path = "../../libsql"}
 tokio = { version = "1.29.1", features = [ "rt-multi-thread" ] }
 hyper-rustls = { version = "0.25", features = ["webpki-roots"]}
+
+[profile.release]
+strip = "symbols"
```

Comment:
This is something we need to talk about. Removing symbols makes debugging really hard. @penberg @MarinPostma @LucioFranco do you have any opinion about this?

Reply:
Tried this to reduce the size of the binary, Rust binaries are way to large for mobile targets. Currently with symbol stripping the binary sits at 60mbs, which is way too much.

Recently wrote more about this:

https://ospfranco.com/rust-reduce-binary-size/

I can remove it, but don't have the time to see how to apply stripping and lto optimization for conditional targets, did not find any documentation on how to do it.

Reply:
Then maybe comment it out with a note that it should be uncommented for android/ios builds to reduce the size of the binary

Reply:
this is biting us in production already, I'm going to remove it. If you need stripped binaries, I think it's easy enough to call `strip` on your binary?

Reply:
It's already removed, I commented it out